### PR TITLE
feat(Tag): add new component

### PR DESCRIPTION
**Pull request by thaytharma**
Added a new `Tag` component as requested by several teams.
This is an initial and basic version as we want to include a clickable tag with `onClick`prop.

Version history:
 - Add `text` / `children` prop
 - Add `icon` prop
 - Add default `Space` and `skeleton` props
 - Add portal site for `Tag`
 - Add `Tag` to sandbox
 - Add tests for `Tag`

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/components/tag/info.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+showTabs: true
+---
+
+## Description
+
+The Tag is a compact component for displaying represent discrete information.<br/>
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/tag/info.md:7]:
```suggestion
The Tag is a compact component for displaying discrete information.<br/>
```

File: packages/dnb-eufemia/src/components/tag/__tests__/Tag.screenshot.test.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  })
+
+  it('have to match Tag with secondary icon', async () => {
+    const screenshot = await testPageScreenshot({
+      selector: '[data-visual-test=""tag-secondary-icon"] .dnb-tag',
+    })
+    expect(screenshot).toMatchImageSnapshot()
+  })
+
+  it('have to match Breadcrumb multiple with children', async () => {
```
Comment (langz) [inline packages/dnb-eufemia/src/components/tag/__tests__/Tag.screenshot.test.js:34]: Seems like we've forgotten to remove a few tests from the Breadcrumb 🙏

**Review by langz — COMMENTED** [review]
Looks very good, nice job 👍

File: packages/dnb-eufemia/src/components/tag/__tests__/Tag.screenshot.test.js (hunk 2 above)
Comment (thaytharma) [reply packages/dnb-eufemia/src/components/tag/__tests__/Tag.screenshot.test.js]: You're right, updated!

File: packages/dnb-design-system-portal/src/docs/uilib/components/tag/info.md (hunk 1 above)
Comment (thaytharma) [reply packages/dnb-design-system-portal/src/docs/uilib/components/tag/info.md]: Updated 😊

File: packages/dnb-eufemia/src/components/tag/Tag.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      data-testid="tag"
+      {...props}
+    >
+      {icon && (
+        <span data-testid="tag-icon" className="dnb-tag__icon">
+          <Icon icon={icon} right="x-small" />
+        </span>
+      )}
+      {content && (
+        <>
```
Comment (langz) [inline packages/dnb-eufemia/src/components/tag/Tag.tsx:87]: I don't think we need to wrap this span in a fragment 🔧

**Review by langz — APPROVED** [review]
I am super happy with what we have so far 👏 
We should perhaps include `onClick` property before releasing it, so that corporate banking could start using this component as well 👍 

https://dnb-it.slack.com/archives/CMXABCHEY/p1638788050107700?thread_ts=1638536031.100500&cid=CMXABCHEY

That we can do in a future PR though 👍 

Would also like for @tujoworker and/or @dinarosv to look through this PR 🙏

**Comment by langz** [comment]
Seems like when pressing the Events-tab in https://eufemia-feattag.gtsb.io/uilib/components/tag that it redirects to https://eufemia-feattag.gtsb.io/uilib/components/tag/events

The following happens locally:
https://user-images.githubusercontent.com/1359205/146153345-a5303acc-6d60-4357-9a66-b609be668879.mov

File: packages/dnb-eufemia/src/components/tag/Tag.tsx (hunk 3 above)
Comment (thaytharma) [reply packages/dnb-eufemia/src/components/tag/Tag.tsx]: Yes! We need a clickable tag in several apps (in pm netbank as well)

File: packages/dnb-eufemia/src/components/tag/__tests__/Tag.screenshot.test.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  it('have to match Tag with primary icon', async () => {
+    const screenshot = await testPageScreenshot({
+      selector: '[data-visual-test="tag-icon"] .dnb-tag',
+    })
+    expect(screenshot).toMatchImageSnapshot()
+  })
+
+  it('have to match Tag with secondary icon', async () => {
+    const screenshot = await testPageScreenshot({
+      selector: '[data-visual-test=""tag-secondary-icon"] .dnb-tag',
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/tag/__tests__/Tag.screenshot.test.js:29]: Having two `""` fails the visual test 🐛

File: packages/dnb-eufemia/src/components/tag/style/dnb-tag.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+*
+*/
+
+@import '../../../style/components/imports.scss';
+
+.dnb-tag {
+  @include componentReset();
+}
+
+@import './tag.scss';
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/components/tag/style/dnb-tag.scss:12]: I think we need an underline here `./_tag.scss`

**Review by tujoworker — APPROVED** [review]
Awesome work 🎉

File: packages/dnb-eufemia/src/components/tag/__tests__/Tag.test.tsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+import { axeComponent, loadScss } from '../../../core/jest/jestSetup'
+import { Provider } from '../../../shared'
+
+describe('Tag', () => {
+  it('renders without properties', () => {
+    render(<Tag />)
+
+    expect(screen.queryByTestId('tag')).not.toBeNull()
+  })
+})
```
Comment (dinarosv) [inline packages/dnb-eufemia/src/components/tag/__tests__/Tag.test.tsx:13]: Should the tests below also inside this "describe" block? Or do they stand alone for a purpose?

**Review by dinarosv — APPROVED** [review]
Love this! Great work 🙌

File: packages/dnb-eufemia/src/components/tag/__tests__/Tag.screenshot.test.js (hunk 4 above)
Comment (thaytharma) [reply packages/dnb-eufemia/src/components/tag/__tests__/Tag.screenshot.test.js]: Thanks, updated! 😊

File: packages/dnb-eufemia/src/components/tag/style/dnb-tag.scss (hunk 5 above)
Comment (thaytharma) [reply packages/dnb-eufemia/src/components/tag/style/dnb-tag.scss]: Updated! 😊

File: packages/dnb-eufemia/src/components/tag/__tests__/Tag.test.tsx (hunk 6 above)
Comment (thaytharma) [reply packages/dnb-eufemia/src/components/tag/__tests__/Tag.test.tsx]: Fixed, thanks!

**Pull request merged**